CMakeLists.txt

```diff
@@ -56,6 +56,8 @@ if(Boost_FOUND)
 endif()
 
 FIND_PACKAGE(CURL REQUIRED)
+FIND_PACKAGE(OpenSSL REQUIRED)
```

Comment:
We have standardized on lowercase cmake. Please fix these two lines.

Reply:
ok